test/test-helpers/mocked-context.ts

```diff
@@ -58,11 +74,14 @@ export function mockedContext(params?: MockedContextParams): Context<Env> {
             },
           },
           validateAuthenticationCode: {
-            mutate: async (code) => {
+            mutate: async (input: ValidateAuthenticationCodeParams) => {
+              const { code, email, tenantId } = input
               if (code === '000000') {
                 throw new InvalidCodeError();
               }
 
+              caller.validateAuthenticationCode
```

Comment:
this doesn't seem like executable code :smile:

Reply:
hmm.. it should be awaited right?

Reply:
Ahh, that's a leftover from testing with the typescript Params.. I'll remove it